src/main/java/org/cirdles/topsoil/app/utils/SVGSaver.java

```diff
@@ -37,32 +42,64 @@
     public static final String SVG_DOCTYPE_PUBLIC = "-//W3C//DTD SVG 1.1//EN";
     private static final String SVG_DOCTYPE_SYSTEM = "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd";
 
+    /**
+     * Saves the vector image in Topsoil with the .svg file extension
+     *
+     * @param svgDocument   Document to be saved
+     */
     public void save(Document svgDocument) {
+
+        // If there is no file specified, then there is nothing to do.
+        // Otherwise, write the file with the SVG format.
+        generateSaveUI(svgDocument);
```

Comment:
If this is all that's here, why not just rename the other? Have this method do more.